.ddev/config.yaml

```diff
@@ -25,7 +25,7 @@ web_environment:
     - NUXT_HOST=0.0.0.0
     - NUXT_PORT=3000
     - PHAPP_ENV_MODE=development
-nodejs_version: "18"
+nodejs_version: "22"
```

Comment:
it seems this has some general good improvements, can we make a separate PR to get those in asap first?